target/linux/ar71xx/files/arch/mips/ath79/mach-cf-e316n-v2.c

```diff
@@ -355,6 +408,43 @@ static void __init cf_e320n_v2_setup(void)
 MIPS_MACHINE(ATH79_MACH_CF_E320N_V2, "CF-E320N-V2", "COMFAST CF-E320N v2",
 	     cf_e320n_v2_setup);
 
+static void __init cf_e355ac_common_setup(void)
+{
+	u8 *art = (u8 *) KSEG1ADDR(0x1f011000);
+	/* Disable JTAG, enabling GPIOs 0-3 */
+	/* Configure OBS4 line, for GPIO 4*/
+	ath79_gpio_function_setup(AR934X_GPIO_FUNC_JTAG_DISABLE, 0);
+
+	cf_exxxn_common_setup(0x10000, CF_E355AC_GPIO_XWDT_TRIGGER);
+
+	cf_exxxn_qca953x_eth_setup();
```

Comment:
As you wrote in commit message, your board has only one Ethernet interface available, so you shouldn't use `cf_exxxn_qca953x_eth_setup()` as it initializes both interfaces.